docs/design-proposals/node/Unified-Vertical-Scaling-for-Container-and-VM-Pods.md

```diff
@@ -130,6 +140,15 @@ RestartContainer policy takes precedence over NoRestart policy.
 Additionally, if RestartPolicy is 'Never', ResizePolicy should be set to
 NoRestart in order to pass validation.
 
+It is proposed to allow users to set the MAX resource to better control the behavior of the resource resizing.
+This could be a value or percentage specified by users. prior to this policy setting, the ceiling of resources, such as
+CPU and memory will be set to the allocatable value at the host and rely the proper CGroup to control the VM resource
+consumptions.
+
+TBD:
+Due to different technologies limit different level of memory change size, e.g. Ballooning driver has 4K page
```

Comment:
From the description it seems ballooning has less capability than memory hotplug. What's the memory increase limit for ballooning?  And I'm the under the impression that we are using memory ballooning.

Reply:
this is one of decision we will discuss in the review. Max setting is the high mark.  however, Ballooning is essentially targeted for the hypervisor to "get" memory from the VM guest. let's discuss this in the meeting in details.
Hotplug/unplug memory device should be a good feature for Arktos regardless eventually which technology was used for vertical scaling here.

Reply:
I have updated the "design consideration" session for proposal justifications.